backend/stakepoold/rpc/rpcserver/server.go

```diff
@@ -243,3 +243,21 @@ func (s *stakepooldServer) ValidateAddress(ctx context.Context, req *pb.Validate
 		PubKeyAddr: response.PubKeyAddr,
 	}, nil
 }
+
+// WalletMasterPubKeys performs the gRPC command WalletMasterPubKeys.
```

Comment:
Maybe I don't need this comment? I see dcrwallet doesn't comment these. 
https://github.com/decred/dcrwallet/blob/master/rpc/rpcserver/server.go

Reply:
I took it out.